### ISSUE TITLE
restore explanation of TEI namespace

### ISSUE THREAD

**Issue by lb42** [TEIC/TEI#1871]
As any fule kno,  http://www.tei-c.org/ns/1.0 is a namespace URI and theref=
ore doesn't necessarily take you anywhere on the web. Unfortunately that do=
esn't stop beginners clicking on it and expecting something helpful to appe=
ar.

At the moment it takes you to a page saying "Oops! That page can=92t be fou=
nd."

Could it not take you to a more helpful page explaining that this is the ho=
me of the TEI namespace which doesn't necessarily blah blah ...?

**Comment by peterstadler** [comment]
This *used to* work, see #1592 

The fix was done outside Wordpress, see last [comment by @kshawkin ](https://github.com/TEIC/TEI/issues/1592#issuecomment-392554638)
> Since the desired URLs, without trailing slashes, couldn't be done in WordPress either, I decided to handle this outside of WordPress in Apache. Our sysadmin has helped me fix these, so you should now see the desired rendering for both:

> http://www.tei-c.org/ns/1.0
> http://www.tei-c.org/ns/Examples

**Comment by peterstadler** [comment]
I think @ldmm is the only(?) one able to fix this?

**Comment by jamescummings** [comment]
Best solution is to have http://tei-c.org/ns/1.0 (which currently redirects to non-existent https://tei-c.org/ns/1.0) instead redirect to something that clearly not the namespace but describes it. e.g. https://www.tei-c.org/ns/documention which describes all of our namespaces (so the examples one as well would redirect there).

**Comment by hcayless** [comment]
I have added pages for the namespaces in Wordpress, and it seems to work. We should check the wording and decide whether these are satisfactory.

**Comment by lb42** [comment]
Wot is wordpress?

**Comment by hcayless** [comment]
@lb42 The content management system that runs the current TEI website.

**Comment by sydb** [comment]
F2F agrees (over at least one objection) that the NS explanation page URL, i.e. the URL to which one is redirected if one follows the TEI NS itself, should be changed to something that is obviously not a namespace. Ticket is green for assignees to come up with a name and suggest it to Council.

We also note that the prose of the page should be re-written. @ebeshero is planning to do that.

**Comment by raffazizzi** [comment]
F2F lunch subgroup suggests the new url to be:

tei-c.org/about-namespaces

**Comment by lb42** [comment]
Not aboutNamespaces?

reluctantly using Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by jamescummings** [comment]
No, because these aren't TEI elements or anything and it is more standard to hyphenate in wordpress sites. (So this is more predictable).  As long as it doesn't *look* like the TEI NS so can't be easily confused for it, and discusses all TEI namespaces (and potentially others), then I'm happy.

**Comment by ebeshero** [comment]
@hcayless We're reviewing this quickly while prepping for the Kraków F2F and wondering if this can be dispatched quickly, since this works now on the new website: https://tei-c.org/ns/1.0/ 

* Drafting some new prose for this shouldn't be too hard...either of us could do it. 
* Do we still want the https://www.tei-c.org/about-namespaces proposed on the ticket?

**Comment by hcayless** [comment]
I don't know that I see the point in a separate about page. I'm in favor of closing it.
